python/pyspark/ml/feature.py

```diff
@@ -931,6 +937,63 @@ class StringIndexerModel(JavaModel):
     """
     Model fitted by StringIndexer.
     """
+    @property
+    def labels(self):
```

Comment:
copy Scala doc: "Ordered list of labels, corresponding to indices to be assigned"